lavorare-per-i-clienti/consuntivazione.md

```diff
@@ -38,21 +38,25 @@ Non include:
 - Tempo di studio non direttamente e non esclusivamente correlato con il lavoro che si sta svolgendo
 - Tempo di pausa lunghi (>30 minuti al giorno)
 
-Come potete immaginare esistono diverse zone d'ombra nel confine tra attivita incluse e attività non 
-incluse. La raccomandazione è quella di usare il buonsenso e di chiedere una mano ad un'altra 
-persona, magari che è sul progetto con voi, se vi trovate in difficoltà nella determinazione del 
+Come potete immaginare esistono diverse zone d'ombra nel confine tra attivita incluse e attività non
+incluse. La raccomandazione è quella di usare il buonsenso e di chiedere una mano ad un'altra
+persona, magari che è sul progetto con voi, se vi trovate in difficoltà nella determinazione del
 numero di ore svolte.
 
-Un piccolo trucco che potete utilizzare è cercare di quantificare le attività non incluse e 
+Un piccolo trucco che potete utilizzare è cercare di quantificare le attività non incluse e
 sottrarle alle ore di presenza in ufficio.
 
-Un caso particolare: I tempi di attesa delle risposte del cliente che passo girandomi i pollici 
-devono essere inclusi? La risposta è: non trovarsi mai nella situazione di doversi girare i pollici 
-e trovare qualcosa da fare comunque. Se proprio non si ha nulla da fare questo tempo NON va incluso 
-nel "tempo effettivamente svolto" ma ne va data pronta comunicazione al cliente, segnalando 
+Un caso particolare: I tempi di attesa delle risposte del cliente che passo girandomi i pollici
+devono essere inclusi? La risposta è: non trovarsi mai nella situazione di doversi girare i pollici
+e trovare qualcosa da fare comunque. Se proprio non si ha nulla da fare questo tempo NON va incluso
+nel "tempo effettivamente svolto" ma ne va data pronta comunicazione al cliente, segnalando
 l'inefficienza.
 
 Per segnare su Float i tempi di pausa e i tempi di attesa utilizziamo il progetto `Time-Wasting`.
 
-La consuntivazione delle ore deve avvenire a fine giornata sugli strumenti previsti (Float e 
-registro del cliente se presente). 
+La consuntivazione delle ore deve avvenire a fine giornata sugli strumenti previsti (Float e
+registro del cliente se presente).
+
+Ricorda che per qualsiasi motivo non viene fatta una giornata concordata su un progetto dovrai
```

Comment:
"Ricorda che, se per qualsiasi motivo non dovessi poter lavorare sul progetto di un cliente in un giorno della settimana, dovrai recuperare la giornata di venerdì."?